devicetypes/smartthings/ecolink-wireless-siren.src/ecolink-wireless-siren.groovy

```diff
@@ -74,6 +74,9 @@ def parse(String description) {
 }
 
 def zwaveEvent(physicalgraph.zwave.commands.basicv1.BasicReport cmd) {	
+	if(cmd.value == 0) {
```

Comment:
Can you add a comment that explains why this is done.

Reply:
comment added